docs/tutorials/projects/verification-system.md

```diff
@@ -0,0 +1,241 @@
+---
+title: "Create a Facial Verification System"
+linkTitle: "Verification System"
+type: "docs"
+description: "Configure a verification system module to implement a facial verification security system on your smart machine."
+image: "/tutorials/verification-system/disarmed.png"
+imageAlt: "Bijan disarming the facial verification system."
+images: ["/tutorials/verification-system/disarmed.png"]
+tags: ["mlmodel", "vision", "services", "security", "camera"]
+authors: ["Sierra G."]
+languages: []
+viamresources: ["mlmodel", "vision", "camera"]
+level: "Intermediate"
+date: "28 November 2023"
+cost: "0"
+no_list: true
+# SMEs: Bijan Haney
+---
+
+Follow this tutorial to create your own machine learning (ML) enabled security system, or, "verification system."
+The verification system configuration you will use in this project combines two detectors: The first detector detects whether there are people present, thanks to a person-detection ML model you train yourself.
+You give the second detector photos of known faces and it tells you if a person is recognized.
+
+The verification system has three possible layers of detectors.
+You layer them together so that objects detected in the images your verification's system camera captures point to countdown, alarmed, or disarmed states.
+With our recommended combination of detectors, your verification system will trigger a countdown when it detects people and disarm the alarm if it detects your face with the facial detector within the countdown period.
+
+For example:
+
+{{<video webm_src="/tutorials/verification-system/demo.webm" mp4_src="/tutorials/verification-system/demo.mp4" alt="Bijan interacting with the verification system" poster="/tutorials/verification-system/trigger_1.png">}}
+
+## Requirements
+
+- A [camera](/components/camera/)
+- A computer running `viam-server` that the camera can connect to, like a [single-board computer (SBC)](/components/board/)
+
+Before configuring your camera, you must [create a robot](/fleet/machines/#add-a-new-robot).
+
+1. On the locations page of the [Viam app](https://app.viam.com), add a new robot by providing a name in the **New Robot** field and clicking **Add robot**.
+2. Click on that robot's name to go to its page.
+
+## Configure a camera
+
+Navigate to the **Config** tab of your machine's page on the [Viam app](https://app.viam.com).
+Configure the camera you want to use for your security system.
+
+We configured ours as a webcam named `"my-webcam"`.
+Follow [these instructions](/components/camera/webcam/) to do so.
```

Comment:
Please copy the relevant bits here so that folks don't have to jump around the docs?

Reply:
I don't know if it's wise to put the full attribute table in as it's pretty huge with optional attributes, but added more stepwise information.